src/main/java/offeneBibel/osisExporter/ObOsisGeneratorVisitor.java

```diff
@@ -137,17 +137,33 @@ else if(node.getNodeType() == ObAstNode.NodeType.quote) {
             if(m_quoteCounter>0)
             {
                 m_quoteCounter++;
-                m_currentFassung.append("»<q level=\"" + m_quoteCounter + "\" marker=\"\"" + end);
+                
+                m_currentFassung.append("»");
+                
+                if (node.getParent().isDescendantOf(ObAstNode.NodeType.italics))
+                {
+                    m_currentFassung.append("</hi>");          
```

Comment:
You close the <hi> block when a quote starts. But you don't open it again after the quote stops. That causes a dangling </hi>, doesn't it?